lib/live_cluster/show_controller.py

```diff
@@ -1287,6 +1289,35 @@ async def _do_default(self, line):
         return self.view.show_users(resp, **self.mods)
 
 
+@CommandHelp(
+    "Displays users, open connections, and quota usage",
+    "for the Aerospike cluster.",
+    "Usage: users [user]",
+    "  user          - Display output for a single user.",
+)
+class ShowUsersStatsController(LiveClusterCommandController):
+    def __init__(self):
+        self.getter = GetAclController(self.cluster)
+
+    async def _do_default(self, line):
+        user = None
+
+        if line:
+            user = line.pop(0)
+
+        users_data = None
+
+        if user is None:
+            users_data = await self.getter.get_users(nodes=self.nodes)
+        else:
+            users_data = await self.getter.get_user(user, nodes=self.nodes)
+
+        if all([isinstance(data, Exception) for data in users_data.values()]):
```

Comment:
raise an exception if all are an exception. Otherwise display the table with some `~`